ods_ci/utils/scripts/rosa/rosa.py

```diff
@@ -0,0 +1,121 @@
+import argparse,sys,os
+dir_path = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(dir_path + "/../")
+from logger import log
+from awsOps import aws_configure
+from rosaOps import (
+    rosa_create_cluster, 
+    create_account_roles,
+    wait_for_osd_cluster_to_be_ready
+    )
+
+
+
+class RosaClusterManager:
+    def __init__(self, args={}):
+        self.aws_access_key_id = args.get("aws_access_key_id")
+        self.aws_secret_access_key = args.get("aws_secret_access_key")
+        self.aws_region = args.get("aws_region")
+        self.profile = args.get("profile")
+        self.cluster_name = args.get("cluster_name")
+        self.compute_nodes = args.get("compute_nodes")
+        self.compute_machine_type =args.get("compute_machine_type")
+        self.rosa_version = args.get("rosa_version")
+        self.channel_name  = args.get("channel_name")
+
+    def create_rosa_cluster(self):
+        print( self.cluster_name, self.aws_region ,self.channel_name, self.compute_nodes,
+              self.compute_machine_type, self.rosa_version)
+        aws_configure(self.aws_access_key_id, self.aws_secret_access_key, self.aws_region)
+        create_account_roles()
```

Comment:
My understanding is that creating account roles is just a one-time task. That means that it is not necessary to run it for each time.

Reply:
we have approx 9 slave i guess its it will not affect the CI if we run every time

Reply:
We could add a check if already added then we can re use it. Not urgent, you can take it as an enhancement.